Automattic/woocommerce-payments
# Add sorting to Disputes list

Fixes #3622 

#### Changes proposed in this Pull Request


This PR adds sorting capabilities to the Disputes list, in the same way as it is already implemented in other list views like Transactions or Deposits. The Disputes data can be sorted by clicking on any of the following table headers:

- Disputed on (by default)
- Amount
- Respond by


<img width="1562" alt="Screen Shot 2022-01-13 at 16 49 58" src="https://user-images.githubusercontent.com/1553182/149389621-507ea3b6-3ca8-4c9e-ad0d-263c985c99c5.png">



#### Testing instructions





* Navigate to Disputes list
* Click on any of sortable table headers
* Observe the data is sorted in ascending or descending order

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](http
s://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : [Testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-3.7.0#introduce-sorting-on-disputes-page)
